lib/datasource/hex/index.spec.ts

```diff
@@ -7,11 +7,12 @@ import { HexDatasource } from '.';
 
 const hostRules: any = _hostRules;
 
-const res1 = loadJsonFixture('certifi.json');
+const certifiResponse = loadJsonFixture('certifi.json');
+const privatePackageResponse = loadJsonFixture('private_package.json');
 
 jest.mock('../../util/host-rules');
 
-const baseUrl = 'https://hex.pm/api/packages/';
+const baseUrl = 'https://hex.pm/api';
```

Comment:
It's unfortunate that the string `/packages` now is repeated nearly a dozen times.

Reply:
That's true - my thinking was that `https://hex.pm/api/packages/` is not really a base URL for _all_ API calls. Actually, `httpMock.scope(baseUrl)` could be extracted as a mocked API client to avoid duplication. WDYT?

Reply:
This change is ok.